magento/pwa-studio
# [feature] intermediate breadcrumbs

## Description

In 2.3.3 we were not able to retrieve the path for intermediate level breadcrumbs. 2.3.4 allows this.

## Related Issue





Closes [PWA-161](https://jira.corp.magento.com/browse/PWA-161).

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Go to a product page and click an intermediate breadcrumb (not Home or the last category. it should link to the right category.
2. Regress breadcrumbs and make sure nothing is broken.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
